R/util.R

```diff
@@ -21,6 +21,14 @@ read_json <- function(path) {
 }
 
 
+read_string_as_json <- function(path) {
```

Comment:
Because the docs are fairly long blocks of html, it seemed easiest to store/edit them as html files. To read them in as valid json strings then need to wrap the whole text in quotes. Seem like a reasonable approach?

Reply:
Does this require any escaping of embedded quotes? We might need a `gsub('"', '\\"', str)` too? Failing that, we might add a check to ensure that they do not contain any embedded quotes (`grepl('"', str, fixed = TRUE)`) and throw an error.

Somewhat tidier might be to have this endpoint return html but that would require a porcelain change which feels like yakshaving here.